model/LtiAssignment.php

```diff
@@ -119,15 +119,11 @@ protected function verifyToken(KernelResource $delivery, User $user)
 
     private function verifyAvailabilityFrame(KernelResource $delivery): void
     {
-        [[$scheduledStartTimeProperty], [$scheduledEndTimeProperty]] = array_values($delivery->getPropertiesValues(
```

Comment:
If it was up to me, I would've probably just added the error-control operator before the array destructuring. It's a good use case for that operator IMO, as it won't be suppressing any function or method warning/error. Usage example [here](https://3v4l.org/EqrJQ).
```
@[[$scheduledStartTimeProperty], [$scheduledEndTimeProperty]] = array_values($delivery->getPropertiesValues(
...
```

Two `getOnePropertyValue()` essentially mean two `getPropertiesValues()` calls under the hood.

Reply:
Thanks for the suggestion! I have just updated the PR :)